utils/fetch-retry-and-confirm-version.ts

```diff
@@ -1,8 +1,8 @@
 import arg from 'arg'
 import originalFetch from 'node-fetch'
 
-function getExpectedData(prismaVersion: string) {
-  return `{"version":"${prismaVersion}","createUser":{"id":"12345","email":"alice@prisma.io","name":"Alice"},"updateUser":{"id":"12345","email":"bob@prisma.io","name":"Bob"},"users":{"id":"12345","email":"bob@prisma.io","name":"Bob"},"deleteManyUsers":{"count":1}}`
+function getExpectedData(prismaVersion: string, binaryString = "") {
+  return `{"version":"${prismaVersion}","createUser":{"id":"12345","email":"alice@prisma.io","name":"Alice"},"updateUser":{"id":"12345","email":"bob@prisma.io","name":"Bob"},"users":{"id":"12345","email":"bob@prisma.io","name":"Bob"},"deleteManyUsers":{"count":1}${binaryString}}`
```

Comment:
>This modifies the check script to also expect and compare a "binaryString" that is basically a list of the files in a specific folder in the deployment, node_modules/.prisma/client. Test can then define what files (and binaries) to expect there, and fails if this changes.

Looks like getExpectedData will never fail since it expects the same var? @janpio

Reply:
Isn't that the job of `JSON.stringify(data) !== JSON.stringify(expectedData)` ?

Reply:
Yep, @millsp got that right - this just constructs the string that is expected. (Just piling on the current existing code here... will need to be refactored sooner or later.